db/migrate/20220816065006_changes_in_tables.rb

```diff
@@ -0,0 +1,6 @@
+class ChangesInTables < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
+  end
+end
```

Comment:
why did you add those things in this PR?

Reply:
sir mistake ho gyi

Reply:
remove those changes